### ISSUE TITLE
[a11y] - Contrast of labeled text on button is too low

### ISSUE THREAD

**Issue by ioana-chiorean** [codeeu/codeweek#1587]
<img width="220" alt="Screenshot 2021-10-11 at 10 19 53" src="https://user-images.githubusercontent.com/1391451/136750151-06290c40-c068-44dc-ab0f-ab3152b7569a.png">

<img width="301" alt="Screenshot 2021-10-11 at 10 33 29" src="https://user-images.githubusercontent.com/1391451/136750302-a3c6410c-4da4-4629-ad5f-a145d2f6ee34.png">

https://www.w3.org/TR/WCAG21/#dfn-contrast-ratio

**Comment by ioana-chiorean** [comment]
@alainvd @AnnikaOP can we make it black maybe? 
I can also do a full accessibility audit or try to at least - I think it is very important part of the website!

**Comment by alainvd** [comment]
The buttons have been fixed.

<img width="345" alt="CleanShot 2022-04-05 at 15 41 31@2x" src="https://user-images.githubusercontent.com/3462620/161767311-a2cd256c-bb95-47f6-b669-047be8ed1a87.png">

**Comment by alainvd** [comment]
A full audit would be nice indeed ! Good idea 👍
